src/linxioserver.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Linx TCP IO Server
+After=labview.service
```

Comment:
Is this compatible with LV?  Does this service need to be stopped before you can run VIs that use the LINX API on the target?

Reply:
Yes.  If you tried to use I/O with deployed VIs and run the I/O server at the same time, it would result in undefined behavior.  However, this service doesn't run by default, you have to explicitly start it.

Reply:
Cool, as long as it's off by default that seems safe.